### PR TITLE
Don't display "Add Unknown as contact to send a Message" if chat is not loaded

### LINKED ISSUE #17153: Delayed display of 'Add Unknown as contact to send a Message' text after joining opened community
### Problem:
The text "Add unknown to send a message" is displayed 5-10 seconds after a user has joined an open community when they open the chat immediately after joining.

### Preconditions:
The users are just created

### Steps:
1. `User A` creates an open community and shares it with User B.
2. `User B Android` joins the community.
3. `User B Android` immediately opens the chat.

### Actual result:
The "Add unknown to send a message" text is shown 5-10 seconds after opening the chat.
image
 
### Expected result:
Potential solutions:
- The chat should be opened without displaying the "Add unknown to send a message" text.
- Display a loading skeleton while the chat data is being loaded.
https://www.figma.com/file/PPWkgOYlZZDxZv5SDGsZVV/Posts-%26-Attachments-for-Mobile?type=design&node-id=3071-278296&mode=design&t=kSP2OocDNfuQ5IxV-0

### logs:
[Status.log](https://github.com/status-im/status-mobile/files/12484194/Status.log)
[geth.log](https://github.com/status-im/status-mobile/files/12484195/geth.log)

### PR DESCRIPTION
**Pull request by Parveshdhull**
fixes https://github.com/status-im/status-mobile/issues/17153

status: ready

### CODE REVIEW

**Review by ibrkhalil — APPROVED** [review]

File: src/status_im2/contexts/communities/overview/view.cljs [hunk 1]
```diff
@@ -210,7 +211,7 @@
    (when (and (not locked?) id)
      {:on-press      (fn []
                        (rf/dispatch [:dismiss-keyboard])
-                       (rf/dispatch [:chat/navigate-to-chat (str community-id id)]))
+                       (debounce/dispatch-and-chill [:chat/navigate-to-chat (str community-id id)] 1000))
```
Comment (ibrkhalil) [inline src/status_im2/contexts/communities/overview/view.cljs:214]: Do you think it might be better to use a constant?

**Comment by pavloburykh** [comment]
@Parveshdhull thank you for the PR. 

Seems like "Add Unknown as contact to send a Message" does not appear any more. 

At the same time I have faced again the issue when chat screen has not been loaded. In fact this is the main problem for us. Bug happens randomly that's why there are no reliable reproduction steps. @Parveshdhull can we do something about this issue? 

[logs (11).zip](https://github.com/status-im/status-mobile/files/13068070/logs.11.zip)

https://github.com/status-im/status-mobile/assets/97245802/162146e5-2e12-4589-ae82-7f0e1dc9a318

**Review by alwx — APPROVED** [review]

File: src/status_im2/contexts/communities/overview/view.cljs (hunk 1 above)
Comment (Parveshdhull) [reply src/status_im2/contexts/communities/overview/view.cljs:214]: probably ok to just pass dispatch delay directly, as other instances are doing same.

**Comment by Parveshdhull** [comment]
hi @pavloburykh, Thank you very much for testing the PR.

The only UI-related reason I can think of that can cause this issue is not having chat ID while opening the chat screen.
This might happen if we click on the chat item **quickly** and open the chat screen again which will close the previous chat screen and lead to the removal of the chat ID. 

So to fix this issue, I added the required gap between presses for opening community chat to be at least 1 Sec and this should fix UI related issue.

But from the video, it looks like there is probably another bug, which is in the status-go, and causing chat data to not be returned after joining the community.
I was able to reproduce this issue in PR builds, but in the debug build my delay was only 2-3 seconds, after that data always returned.

### Why problem in status-go, why not UI issue?
1. It doesn't look issue with the chat ID as chat is still not working after reopening(after resetting chatID). Also, other chats are opening without any problem, so the issue is with that particular chat, not the UI.
2. Whenever we open a community from 1-1 to chat, we also change the bottom tab to communities stack. But as you can see in the video, once we open the "general" chat, the tab changes back to chats stack, because db doesn't have that data, so we don't know if the chat is community chat or 1-1.

One thing I can do for now just disable the opening of chat until chat data is loaded from the DB, this should be okay in case of a 2-3-second delay, but in case data is not returned until login probably not helpful.
Or we can merge PR for now and log this issue with status-go separately. wdyt?

**Comment by pavloburykh** [comment]
@Parveshdhull thank you for your investigation. 

> Or we can merge PR for now and log this issue with status-go separately. wdyt?

I prefer this option. We will keep an eye on this issue and in case we will be facing it too often I will log a separate issue. As for now I cannot say that we are facing it too often. But when it happens in our e2e run it results in failing the whole group of tests.

**Comment by pavloburykh** [comment]
If there were no additional changes after my last check - PR is ready to be merged.

**Pull request merged**